Source/WebView/turbo.js

```diff
@@ -185,6 +192,19 @@
         })
       }
     }
+
+    isPageRefreshWithBackwardsCompatibility (location, action) {
```

Comment:
Not sure if we need backwards compatibility for current Turbo 8 versions? Anyway, feel free to remove this if not.